package.json

```diff
@@ -20,20 +20,14 @@
   "bugs": "https://github.com/nosir/cleave.js/issues",
   "homepage": "https://github.com/nosir/cleave.js",
   "docs": "https://github.com/nosir/cleave.js/blob/master/README.md",
-  "main": "src/Cleave.js",
+  "main": "dist/cleave.js",
```

Comment:
I did this on purpose but I don't remember anymore why (didn't work for me). I'll come back to you on this, soon!